File: dog-kitty-backend/DogNKitty/Entity/Animal.cs
using System;
using System.Collections.Generic;
using Utils.Enums;

namespace Entity
{
    public class Animal
    { 
        public int Id { get; set; }
        public int Nome { get; set; }
        public StatusEnum Status { get; set; }
        public List<Foto> Foto { get; set; }
        public double Peso { get; set; }
        public int Idade { get; set; }
        public SexoEnum Sexo { get; set; }
        public PorteEnum Porte { get; set; }
    }
}

File: Kitty-Doggy/src/domain/animal.ts
import { AnimalEnum } from "./enum/animalEnum";
import { PorteEnum } from "./enum/porteEnum";
import { SexoEnum } from "./enum/sexoEnum";
import { StatusEnum } from "./enum/statusEnum";
import { Foto } from "./foto";
import { Raca } from "./raca";

export class Animal {

    Id: number|undefined;
    Nome: string|undefined;
    Status: StatusEnum|undefined;
    Foto: Array<Foto>|undefined; 
    Peso: number|undefined;
    Idade : number|undefined;
    Sexo: SexoEnum |undefined;
    Porte:PorteEnum|undefined;
    TipoAnimal: AnimalEnum |undefined;
    Raca : Raca|undefined;
}
